src/components/ADempiere/Form/VPOS/Order/index.vue

```diff
@@ -123,7 +123,7 @@
                       <el-input-number
                         ref="editField"
                         v-model="scope.row.discount"
-                        v-shortkey="isEditQtyOrdered ? {close: ['esc']} : {}"
+                        v-shortkey="isEditQtyOrdered ? {close: ['esc'], enter: ['enter']} : {}"
```

Comment:
@elsiosanchez please change to computed https://github.com/adempiere/adempiere-vue/blob/develop/src/components/ADempiere/ContextMenu/contextMenuMixin.js#L210

```javascript
  shorcutKey() {
    if (this.isEditQtyOrdered) {
      return {
        close: ['esc'],
        enter: ['enter']
      }
    }
    return {}
  }
```